Week3/homework/step2-7.js

```diff
@@ -18,6 +18,7 @@ function f2(val) {
 
 f2(y);
 
-console.log(y);
+console.log(y); // {x: 10}
 
-// Add your explanation as a comment here
+// x is a constant number. Function creates a block scoped x variable. X can chance in block scope.
+// y is an object. Function can change x key value. Object keys and values mutable.
```

Comment:
I don't think that is exactly what's happening here. As mentioned in the homework, look for call by value vs call by reference in JavaScript. 🙂

Reply:
I will look again